defaults/main.yml

```diff
@@ -11,9 +11,7 @@ galaxy_tools_galaxy_instance_url: 127.0.0.1:8080
 
 # A file containing all the tools to be installed. See `files/tool_list.yaml.sample`
 # file for more about the format requirements of this file. The file names must be unique.
-# TODO: Once Ansible 2.0 comes out, re-enable multiple files here
-# (see https://docs.ansible.com/ansible/playbooks_loops.html#loops-and-includes)
-galaxy_tools_tool_list: "tool_list.yaml"
+galaxy_tools_tool_list_files: [ "tool_list.yaml" ]
```

Comment:
Can you change the comment to read "A list of yml files that define tools to be installed" ?